onoff.js

```diff
@@ -243,4 +248,4 @@ class Gpio {
 }
 
 exports.Gpio = Gpio;
-
+module.exports = { Gpio, GPIO_ONE, GPIO_ZERO };
```

Comment:
I think it would be better if `HIGH` and `LOW` were properties of `Gpio`. If not, code will look a bit strange, for example, this:
```js
const onoff = require('onoff');
const Gpio = onoff.Gpio;

let led = new Gpio(17, 'out');
if (led.readSync() === onoff.HIGH) {
  console.log('The LED is on');
}
```
rather than this:
```js
const Gpio = require('onoff').Gpio;

let led = new Gpio(17, 'out');
if (led.readSync() === Gpio.HIGH) {
  console.log('The LED is on');
}
```